modules/ROOT/pages/enable-and-disable-a-public-portal.adoc

```diff
@@ -0,0 +1,76 @@
+= Enable and Disable a Public Portal
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All organizational public portals in Exchange created after July 2022 are disabled by default. You cannot sign in to view assets or share new assets on your public portal unless you enable them. You must be an Exchange administrator to enable or disable a public portal.
```

Comment:
The word "organizational" is a bit strange considering that there is no "non" organizational public portal 🤔. I know this was originally written by me 😛  but maybe we can delete that word and leave "All public portals in Exchange ...". What do you think?

Reply:
Yes, that makes sense to me. I will change "organizational' to "all".